main.js

```diff
@@ -0,0 +1,29 @@
+const generateNumbers = () => {
```

Comment:
It probably makes more sense for this not to be an arrow function and just a regular named function.